### PR TITLE
Adds 4.7.5 RNs

### PR DESCRIPTION
**Pull request by stevsmit**
RNs 4.7.5 for 3-5. 

Preview at https://deploy-preview-31220--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-5

Do not merge until Monday. Thanks!

### CODE REVIEW

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -2179,3 +2179,48 @@ link:https://access.redhat.com/solutions/5906541[{product-title} 4.7.4 container
...
+
+link:https://access.redhat.com/solutions/5920891[{product-title} 4.7.5 container image list]
+
+[id="ocp-4-7-5-features"]
+==== Features
+[id="ocp-4-7-5-installing-cluster-on-vmc-on-aws"]
+===== Installing a cluster on VMC on AWS
+You can now install an {product-title} cluster on VMware vSphere infrastructure by deploying it to VMware Cloud (VMC) on AWS. See the documentation for xref:../installing/installing_vmc/installing-vmc.adoc#installing-vmc[deploying a cluster to VMC] for more information.
+[id="ocp-4-7-5-adding-memory-and-uptime-metadata-to-IO-acrhive"]
+===== Adding memory and uptime metadata to the IO archive
```
Comment (jeana-redhat) [inline release_notes/ocp-4-7-release-notes.adoc:2200]: I haven't seen Insights Operator abbreviated to IO before - should write it out in the title and body of this enhancement text.

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -2179,3 +2179,48 @@ link:https://access.redhat.com/solutions/5906541[{product-title} 4.7.4 container
...
+This update adds `uptime` and `memory alloc` metadata to the IO archive so that small memory leaks can be investigated properly. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1935605[*BZ#1935605*].
+[id="ocp-4-7-5-SAP-license-management-enhancement"]
+===== SAP license management enhancement
+Use the following command to detect failure in the license management pod:
```
Comment (jeana-redhat) [inline release_notes/ocp-4-7-release-notes.adoc:2204]: Is this command new? Might want to intro with "With this release, you can now..." if so.

**Review by jeana-redhat — COMMENTED** [review]
Just a couple things, output looks good :+1:

File: release_notes/ocp-4-7-release-notes.adoc (hunk 2 above)
Comment (stevsmit) [reply release_notes/ocp-4-7-release-notes.adoc:2204]: Ahhh I had that at first and then removed it. I'll add it back :D

**Pull request merged**